src/rgw/rgw_process.cc

```diff
@@ -245,10 +249,61 @@ int process_request(rgw::sal::RGWRadosStore* const store,
   dout(10) << "op=" << typeid(*op).name() << dendl;
 
   s->op_type = op->get_type();
+  //dout(0) << "XXX: " << s->info.env->exists("HTTP_XXX") << " cache "  << (int)s->user->get_info().cache << " ret= " << ret << dendl;
 
   try {
     ldpp_dout(op, 2) << "verifying requester" << dendl;
     ret = op->verify_requester(auth_registry);
+    RGWEnv test(*(s->info.env));
+    char abc[2];
+    abc[0] = (char)178;
+    abc[1] = '\0';
+    char def[2];
+    def[0] = (char)177;
+    def[1] = '\0';
+    dout(0) << "XXX: " << s->info.env->exists("HTTP_X_AMZ_CACHE") << " cache "  << (int)s->user->get_info().cache << " ret = " << ret << dendl;
```

Comment:
also here--if you want a permanent trace, need to make it clear and put it at an appropriate level (e.g., above 10, most likely)